app/routes/reminders/new.js

```diff
@@ -0,0 +1,8 @@
+// ROUTES
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return this.get('store').createRecord('reminder');
```

Comment:
@marissa27 @tbrandle so this line is going to create a temporary model in the store but its kind of just going ot hang out until you tell it officially commit this data to the database. Check out [this section of the ember-groceries](https://github.com/turingschool-examples/ember-groceries/blob/master/app/components/add-grocery.js) example for a pretty big hint